policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/nsaltc/reduction.yaml

```
@@ -0,0 +1,18 @@
+description: Affordable living tax credit reduction rate in Nova Scotia. 
```

Write descriptions for other variables similarly.
```suggestion
description: Nova Scotia reduces its affordable living tax credit at this rate with respect to adjusted family net income.
```